controllers/apim/ratelimitpolicy_controller.go

```diff
@@ -261,10 +266,11 @@ func (r *RateLimitPolicyReconciler) cleanUpOrphanWASMPlugins(ctx context.Context
 
 	RateLimitStages := []apimv1alpha1.RateLimitStage{apimv1alpha1.RateLimitStagePREAUTH, apimv1alpha1.RateLimitStagePOSTAUTH}
 	for _, gwKey := range notReferencedGatewayKeys {
+		wasmPluginDeleted := false
 		for _, stage := range RateLimitStages {
 			wasmKey := kuadrantistioutils.WASMPluginKey(gwKey, stage)
 
-			wasmplugin := &istioextensionv1alpha3.WasmPlugin{}
```

Comment:
downgrade?

Reply:
then the name should be fixed.  You were planning to do that in another PR?

Reply:
By name here, I mean the name of the import i.e. `istioextensionv1alpha1`. It's already in this PR.